core/modules/proto/testProtocol.cc

```diff
@@ -61,9 +61,15 @@ struct ProtocolFixture : public lsst::qserv::proto::FakeProtocolFixture {
             && (t1.chunkid() == t2.chunkid())
             && (t1.db() == t2.db());
 
-        bool sTablesEq = t1.scantables_size() == t2.scantables_size();
-        for(int i=0; i < t1.scantables_size(); ++i) {
-            sTablesEq = sTablesEq && (t1.scantables(i) == t2.scantables(i));
+        bool sTablesEq = t1.scantable_size() == t2.scantable_size();
+        for(int i=0; i < t1.scantable_size(); ++i) {
+            auto const& sTbl1 = t1.scantable(i);
+            auto const& sTbl2 = t2.scantable(i);
+            bool eq = (sTbl1.db().compare(sTbl2.db()) == 0
```

Comment:
I'd prefer to keep functionality separate from the protobuf messages. I'd like to avoid logic in the program depending on the protocol structure, especially since that is highly likely to change in the near future. I also don't see this being used outside of the unit test.